xds/src/test/java/io/grpc/xds/ServerXdsClientTest.java

```diff
@@ -95,6 +95,7 @@
  * Tests for {@link ServerXdsClient}.
  */
 @RunWith(JUnit4.class)
+@Ignore
```

Comment:
What's happening here? Is it just waiting for #7969 to be merged?

Reply:
Yes. After that merge this whole file is going to go away